docs/deployment/kubernetes.mdx

```diff
@@ -1,5 +1,198 @@
+
+After the beeai-platform becomes ready, it's necessary to configure the LLM provider. We will use the `admin-password` you created earlier and your preferred LLM credentials, for example:
+
+```shell
+kubectl run curlpod --image=curlimages/curl -it --rm --restart=Never -- curl -X PUT \
+  beeai-platform-svc:8333/api/v1/variables \
+  -u beeai-admin:my-secret-password \
+  -H "Content-Type: application/json" \
+  -d '{
+    "env": {
+        "LLM_API_BASE": "https://api.openai.com/v1",
+        "LLM_API_KEY": "sk-...",
+        "LLM_MODEL": "gpt-4o"
+    }
+  }'
+```
+
+## Use the Platform
+
+Test that the platform is working:
+
+```shell
+# port-forward in a separate terminal
+kubectl port-forward svc/beeai-platform-svc 8333:8333 &
+```
+
+```shell
+beeai list
+beeai run chat hi
+```
+
+## Configuration Options
+
+### Security Settings
+
+For production deployments, ensure proper security:
+
+```yaml
+# config.yaml
+auth:
+  enabled: true
+  admin_password: "use-a-strong-password-here"
+
+# Generate a secure encryption key
+encryptionKey: "your-32-byte-base64-encoded-key"
+```
+
+For development/testing environments:
+
+```yaml
+# config.yaml - INSECURE, for testing only
+auth:
+  enabled: false
+```
+
+### Agent Configuration
+
+Include specific agents in your deployment:
+
+```yaml
+# config.yaml
+providers:
+  # Community agents
+  - location: ghcr.io/i-am-bee/beeai-platform/community/aider:latest
+  - location: ghcr.io/i-am-bee/beeai-platform/community/gpt-researcher:latest
+  
+  # Official agents
+  - location: ghcr.io/i-am-bee/beeai-platform/official/beeai-framework/chat:latest
+  
+  # Your custom agents
+  - location: your-registry.com/your-team/custom-agent:v1.0.0
+```
+
+### External Agent Registry
+
+Use the default agent registry instead of specifying individual agents:
+
+```yaml
+# config.yaml
+externalRegistries:
+  public_github: "https://github.com/i-am-bee/beeai-platform@release-v0.2.0#path=agent-registry.yaml"
+
+# Omit the 'providers' section to use only registry agents
+```
+
+## Management Commands
+
+### Upgrading
+
+To upgrade to a newer version of the beeai platform, use:
+
+```shell
+helm upgrade --install -f config.yaml beeai oci://ghcr.io/i-am-bee/beeai-platform/beeai-platform-chart/beeai-platform:<newer-version>
+```
+
+### View Current Configuration
+
+```bash
+helm get values beeai
+```
+
+### Check Deployment Status
+
+```bash
+helm status beeai
+kubectl get pods
+kubectl logs deployment/beeai-platform
+```
+
+### Uninstall
+
+```bash
+helm uninstall beeai
+```
+
+## Troubleshooting
+
+### Common Issues
+
+**Platform not starting:**
+```bash
+# Check pod logs
+kubectl logs deployment/beeai-platform
+
+# Check events
+kubectl get events --sort-by=.lastTimestamp
+```
+
+**Can't access agents:**
+```bash
+# Verify LLM configuration
+kubectl logs deployment/beeai-platform | grep -i llm
+
+# Test LLM connectivity from within cluster
+kubectl run test-curl --image=curlimages/curl -it --rm --restart=Never -- \
```

Comment:
this is not necessary, it only tests that the cluster has access to the internet, which is almost always the case :slightly_smiling_face: You probably saw this when debugging [this issue](https://github.com/i-am-bee/beeai-platform/discussions/742), but there I tested host.docker.internal to check whether the cluster has access to host, that's only applicable for local installation